### PR TITLE
UI Enhancements

### PR DESCRIPTION
**Pull request by wbprice**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -105,6 +104,7 @@ <h1 id="title">HRT Bus Finder</h1>
 		<div class="mapcanvas"></div>
 		<div class="extended-info" style="display:none;">
 			<span>Location of <strong>Bus <%= busId %></strong> as of <strong><span class='lastUpdate'><%= lastUpdate %></span></strong>.</span>
+			<p class="arrival-time">Projected Arrival: <strong><%= arriveTime %></strong> - <%= adherence %></p>
```
Comment (bschoenfeld) [inline index.html]: This isn't really the projected arrival. It's the scheduled arrival time. The adherence is how close it is to that scheduled time.

File: css/busfinder.css [hunk 2]
```diff
@@ -256,6 +256,10 @@ h1, h2, h3, h4, h5, p, a {
 	border: 3px solid #DDC6AA;
 }
 
+.imminent {
+	box-shadow: 0em 0em 1em black;
```
Comment (bschoenfeld) [inline css/busfinder.css:260]: What does it look like when there are multiple imminent arrivals?

Comment (wbprice) [inline css/busfinder.css:260]: It be worthwhile to look at only applying .imminent to the bus that's going to arrive the soonest if there are several buses arriving at a given station within a given time frame... or would that never happen?

Comment (bschoenfeld) [inline css/busfinder.css:260]: It definitely happens. Check this [stop out](http://hrtb.us/#/stops/454). I count 5 buses scheduled to arrive in a 2 minute span.

Comment (wbprice) [inline css/busfinder.css:260]: Interesting.  In that case, I'm going to say we only apply the class to the "most imminent" bus and leave the rest as it.  (or just remove it altogether.)

**Pull request merged**